### ISSUE TITLE
Update of the Volla Mesages app not possible (VollaPhone Quintus)

### ISSUE THREAD

**Issue by tux4us** [HelloVolla/volla-os-beta-test#194]
Device: VollaPhone Quintus
OS: 14-20241021-STABLE-algiz
F-Droid Version: 1.20.1
VollaMessages Version: 0.7.0-beta-rc.3

Description:

The current version of VollaMessages on GitHub is version 0.7.3.

Since this app is delivered as a system app on the VollaPhone Quintus, an update via GitHub is not possible and leads to a version conflict.

If I try to update the package sources in F-Droid, I get the message 

`Holochain Nightly Repository: No attributes for index-v1.json...`
![Screenshot_20241125-171902_F-Droid_Error](https://github.com/user-attachments/assets/b18cf0c8-8c45-44d7-939c-b8017384748d)

It is therefore not possible to update the system app. This behaviour is confirmed by other users with VollaQuintus.

**Comment by wurzer** [comment]
Yes, the app on Volla OS has a different signature and mus be updated from F-Droid.

**Comment by tux4us** [comment]
Unfortunately, an update via F-Droid is not possible because updating the package sources results in the error shown above.

**Comment by wurzer** [comment]
You need to update the app from F-Droid, becuase we use a different signature in Volla OS. The necessary repository will be introduced with an update. Further more, the app will be updated with the November system update.

**Comment by franco-bez** [comment]
The Update via F-droid is not possible as the repositories are obviously damaged.
In F-droid scan the Barcode shown here:
![Auswahl_824](https://github.com/user-attachments/assets/02dae377-ce31-4597-837b-0943be245ee9)

https://holochain-fdroid.volla.tech/fdroid/repo/
https://holochain-fdroid-nightly.volla.tech/fdroid/repo/
and you will get an error message on any Volla Device "Wrong Fingerprint - org.fdroid.index.SigningException"

EDIT: 2024.12.12 the signing key error is fixed
But if you try to add both repos to fdroid they are treated as "mirrors".

**Comment by wurzer** [comment]
Fixed in December/January update
